packages/react/src/components/Lightbox/README.md

```diff
@@ -0,0 +1,104 @@
+# Lightbox
+
+> The Lightbox component is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LightBox } from '@carbon/ibmdotcom-react';
+import '@carbon/ibmdotcom-styles/scss/components/lightbox/_lightbox.scss';
+import 'yourapplication.scss';
+
+function App() {
+  return;
+  <LightBox
+    title='Curabitur malesuada varius mi eu posuere'
+    copy='Lorem ipsum dolor sit amet, consectetur adipiscing Aenean et ultricies est.'
+    image={
+        uri: {
+          sm: 'https://via.placeholder.com/640x320',
+          md: 'https://via.placeholder.com/768x384',
+          lg: 'https://via.placeholder.com/1024x512',
+        },
+        alt: 'Placeholder Image',
+      }
+    open={true}
+    onClose={() => alert('<<< model closed >>>')}
+  />;
+}
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+> Don't forget to import the Lightbox styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+#### Feature Flags
+
+To utilize the following features, set the following variable's to `true` within
+your `.env` file or your application build settings.
+
+```
+DDS_LIGHTBOX=true
+```
+
+> See
+> [feature-flags.md](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/docs/feature-flags.md)
+> and
+> [.env.example](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/patterns-react/.env.example)
+> for more information
+
+## Props
+
+| Name      | Required | Data Type | Default Value | Description                                             |
+| --------- | -------- | --------- | ------------- | ------------------------------------------------------- |
+| `title`   | NO       | String    | null          | Title text for lightbox image                           |
+| `copy`    | NO       | String    | null          | Short description text for lightbox image               |
+| `image`   | YES      | Object    | null          | Image object for lightbox. see `Image` below for detail |
+| `open`    | YES      | Boolean   | null          | sets `true | false` whether the lightbox is open/close  |
+| `onClose` | NO       | Function  | null          | Run the function to do something on close               |
+
+### Image
+
+| Name  | Data Type | Description                 |
+| ----- | --------- | --------------------------- |
+| `uri` | object    | Urls for images             |
```

Comment:
@photodow @sannminwin would this be using the responsive image component? we should align on the prop names.

Reply:
I think that's way to go. Where can I find specs for responsive image component ?

Reply:
@jeffchew yes, that's the intent at the moment.

@sannminwin loosely working on this sprint. Will probably be pushing that particular work into some of the extended days of this sprint.

Reply:
I think this is the issue though where you can track those specs. #767